docs/pool-upgrade-how-to.md

```diff
@@ -0,0 +1,37 @@
+# Pool Upgrade How To
+
+Below there are steps which need to be done to upgrade a pool to a new DCL
+application version:
+
+1. A new release of
+   [distributed-compliance-ledger](https://github.com/zigbee-alliance/distributed-compliance-ledger)
+   project is issued. The code of the new released application version must be
+   augmented by a new upgrade handler which will handle an upgrade to this new
+   version. This upgrade handler must have a unique name which will serve as the
+   name of the upgrade to this new version.
+2. A ledger height not reached yet at which all the nodes in the pool must be
+   upgraded to the new application version is chosen.
+3. SHA-256 or SHA-512 checksums of the new application version binaries for
+   supported platforms are calculated.
```

Comment:
Please add a command example (For Ubuntu) that can be used to calculate the checksum

Reply:
Added.